app-misc/piper/metadata.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+
+<pkgmetadata>
+  <maintainer type="person">
+    <email>alex@1stleg.com</email>
+    <name>Alex Barker</name>
+  </maintainer>
+  <maintainer type="person">
+    <email>petteyg359@gmail.com</email>
+    <name>Gordon Pettey</name>
+  </maintainer>
```

Comment:
Also do you have their consent on being added here?

Reply:
Consent was pending, never heard back so I will remove it.  I can combine them, just figured there are two bugs so two PRs.